### ISSUE TITLE
Invalid syntax on rule example

### ISSUE THREAD

**Issue by blackwood821** [joyent/sdc-fwapi#12]
The `index.md` file shows the following rule as an example:

```
FROM any TO tag www ALLOW tcp port (80 and 443)
```

But when I try to create that rule it gives me a syntax error:
```
$ triton fwrule create "FROM any TO tag www ALLOW tcp port (80 and 443)"
triton fwrule create: error (InvalidParameters): Invalid parameters
    rule: InvalidParameter: Error at character 34: '(', expected: 'WORD', 'all', found: (
```

The following syntax is valid though:

```
$ triton fwrule create "FROM any TO tag www ALLOW tcp (port 80 and port 443)"
Created firewall rule 8eaffb46-d4d3-47c6-88a0-bc24d863b360
```

**Comment by danmcd** [comment]
Will be fixing in an upcoming PR.  I'll add you to it if I can, or cross-reference it here.

**Comment by danmcd** [comment]
@blackwood821 please review #14 to make sure I got it right.

**Comment by blackwood821** [comment]
@danmcd Looks good to me

**Comment by bahamat** [comment]
Fixed in #14
